netdata/netdata
# fix(proc.plugin): handle disabled IPv6

##### Summary

Fixes: #14235

Netdata doesn't create charts for `/proc/net/netstat`, `/proc/net/snmp` metrics if fails to read `/proc/net/snmp6` (IPv6 is disabled).

This PR fixes it by **moving the processing of `/proc/net/snmp6` to a separate function** - that is the only change, don't get confused with the number of changed LOC.

##### Test Plan

- Add `ipv6.disable=1` to `GRUB_CMDLINE_LINUX_DEFAULT`; update grub and reboot server.
- Check "Networking Stack" and "IPv4 Networking" charts.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
